react-events/my-app/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { Header } from './Header';
+import { Image } from './Image';
+import { Caption } from './Caption';
+import { Description } from './Description';
+import { Button } from './Button';
+import './App.css';
+
+const srcs = ['/starry-sky.jpeg', '/cool-kitty.jpg', '/cool-car.jpg'];
```

Comment:
This works, but a better way is to use a single array of imageObjects that looks like this
Const imageObjects = [{src: "/starry-sty.jpeg", caption: "A beautiful Image of Space", description: "This a wonderful sky full of stars!"}...]
Then you can pass props like this <Image src={imageObjects[index].src}/>